hawc/apps/assessment/admin.py

```diff
@@ -140,6 +140,13 @@ def migrate_terms(self, request, queryset):
     migrate_terms.short_description = "Migrate endpoint terms"
 
 
+@admin.register(models.Values)
+class ValuesAdmin(admin.ModelAdmin):
```

Comment:
might be good to select_related the assessment too since as it's currently designed it will need to query each assessment in the table as well